force-app/main/default/applications/standard__LightningSales.app-meta.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomApplication xmlns="http://soap.sforce.com/2006/04/metadata">
+    <actionOverrides>
```

Comment:
@t-gul @bmtemakina because you created a new app, I dont think this app is related to the task. You can exclude this file from PR